solana/models/_sector/dex/goosefx/goosefx_ssl_v2_solana_trades.sql

```diff
@@ -60,36 +61,33 @@
             , sp.call_outer_instruction_index as outer_instruction_index
             , COALESCE(sp.call_inner_instruction_index, 0) as inner_instruction_index
             , sp.call_tx_index as tx_index
-            , COALESCE(tk_2.token_mint_address, cast(null as varchar)) as token_bought_mint_address
-            , COALESCE(tk_1.token_mint_address, cast(null as varchar)) as token_sold_mint_address
-            , trs_2.account_source as token_bought_vault
-            , trs_1.account_destination as token_sold_vault
+            , COALESCE(trs_2.token_mint_address, cast(null as varchar)) as token_bought_mint_address
+            , COALESCE(trs_1.token_mint_address, cast(null as varchar)) as token_sold_mint_address
+            , trs_2.from_token_account as token_bought_vault
+            , trs_1.to_token_account as token_sold_vault
         FROM {{ source('goosefx_solana', 'gfx_ssl_v2_call_swap') }} sp
-        INNER JOIN {{ source('spl_token_solana', 'spl_token_call_transfer') }} trs_1 
-            ON trs_1.call_tx_id = sp.call_tx_id 
-            AND trs_1.call_block_time = sp.call_block_time
-            AND trs_1.call_outer_instruction_index = sp.call_outer_instruction_index 
-            AND trs_1.call_inner_instruction_index = COALESCE(sp.call_inner_instruction_index,0) + 1
+        INNER JOIN {{ ref('tokens_solana_transfers') }} trs_1 
+            ON trs_1.tx_id = sp.call_tx_id 
+            AND trs_1.block_time = sp.call_block_time
```

Comment:
```suggestion
            AND trs_1.block_date = CAST(date_trunc('day', sp.call_block_time)
```

Reply:
we should consider always joining to solana transfers on `block_date`, as this is the partition column on that spell and we've seen performance enhancements when doing this

Reply:
this would require introducing `block_date` to all solana dexs -- which we likely should be doing anyway, to match `dex.trades`. 
`block_time`
`block_date`
`block_month`
should be standard across sector spells

Reply:
in order to keep this PR clean, we could take the first step to just use the `CAST` directly in join condition, then follow up with introducing `block_date` and cleaning up downstream joins.
cc @0xRobin